.github/workflows/ubuntu.yml

```diff
@@ -50,12 +54,50 @@ jobs:
       with:
           key: ${{ env.os }}
 
+
+    - name : Init VCPKG submodule
+      run: |
+          git submodule update --init vcpkg
+
+    # Restore both vcpkg and its artifacts from the GitHub cache service.
+    - name: Restore vcpkg and its artifacts.
+      uses: actions/cache@v4
+      with:
+        # The first path is the location of vcpkg (it contains the vcpkg executable and data files).
+        # The other paths starting with '!' are exclusions: they contain termporary files generated during the build of the installed packages.
+        path: |
+          ${{ env.VCPKG_ROOT }}
+          !${{ env.VCPKG_ROOT }}/buildtrees
+          !${{ env.VCPKG_ROOT }}/packages
+          !${{ env.VCPKG_ROOT }}/downloads
+        # The key is composed in a way that it gets properly invalidated: this must happen whenever vcpkg's Git commit id changes, or the list of packages changes. In this case a cache miss must happen and a new entry with a new key with be pushed to GitHub the cache service.
+        # The key includes: hash of the vcpkg.json file, the hash of the vcpkg Git commit id, and the used vcpkg's triplet. The vcpkg's commit id would suffice, but computing an hash out it does not harm.
+        # Note: given a key, the cache content is immutable. If a cache entry has been created improperly, in order the recreate the right content the key must be changed as well, and it must be brand new (i.e. not existing already).
+        key: |
+          ${{ hashFiles( 'vcpkg_manifest/vcpkg.json' ) }}-${{ hashFiles( '.git/modules/vcpkg/HEAD' )}}-${{ env.triplet }}
+
     - name: Install libraries
       run: |
            sudo apt-get update
            sudo apt-get install uuid-dev libwxgtk3.0-gtk3-dev
-           sudo apt-get install libboost-test-dev
            sudo apt-get install g++-10 gcc-10
+          #  sudo apt-get install libyaml-cpp-dev libantlr4-runtime-dev 
+    
+    - name: export wxWidgets script
+      shell: bash
+      run: |
+           echo ${{ env.WX_CONFIG}}
+           cat ${{ env.WX_CONFIG}}
+           export WX_CONFIG=/usr/bin/wx-config
+           cat $WX_CONFIG
```

Comment:
Is this needed ?

Reply:
the env var WX_CONFIG is expected by [FindwxWidgets.cmake](https://github.com/AntaresSimulatorTeam/Antares_Simulator/blob/b515cc9fddf7c2378a8aa7a4b598ab763cd64265/src/cmake/wxWidgets/FindwxWidgets.cmake#L820)

Reply:
It is defined above ⬆️

Reply:
Okay, I'll clean it up